queries.yaml

```diff
@@ -1052,7 +1052,7 @@ post_process:
       end_zoom: 9
       items_matching: { kind: locality }
       max_items: 1
-      grid_width: 3
+      grid_width: 52181.0113
```

Comment:
It's not obvious this is in meters... should the param name be `grid_width_meters`?

Is the fractional meters necessary?

Reply:
I suppose they're not necessary, but using fractions means we get closer to exactly 3x3 subdivisions in a single 256px tile at this zoom level. If we go to integer values here, then the tiny subdivisions at the edge of the tile are slightly bigger and are more likely to get an extra feature in there.

Reply:
Changed to `grid_width_meters` in 2eca280.